### ISSUE TITLE
Create Folder name issue

### ISSUE THREAD

**Issue by Thanatoast** [mate-desktop/caja#447]
Right click in a folder and choose "Create Folder", type in a folder name and then (without pressing enter) double click the folder icon. It takes you to the new directory and shows it as 'untitled folder' rather than the name you just typed. Going back to the sub-dir shows that it did actually name the directory as you entered.
This is Caja 1.8.2 in Linux Mint 17.1.

**Comment by monsta** [comment]
Confirmed in 1.10.3. Never thought anyone would actually do that instead of pressing Enter though :smile:

**Comment by sc0w** [comment]
with nemo 3.4.7 it works fine

**Comment by monsta** [comment]
Nemo had been forked later than Caja, at that time Nautilus might already have the fix.

**Comment by rcaridade145** [comment]
https://bugzilla.gnome.org/show_bug.cgi?id=473981

I believe the fix is under this commit https://gitlab.gnome.org/GNOME/nautilus/commit/6ae3384aa4a0cba83ff803418fb08054980027d8

@sc0w @monsta

**Comment by lukefromdc** [comment]
Starting with the referenced https://gitlab.gnome.org/GNOME/nautilus/commit/6ae3384aa4a0cba83ff803418fb08054980027d8
nautilus actually uses a dialog that pops up when a new folder is created, and as of Nautilus 3.30 it looks like this:

![nautilus_new_folder_dialog](https://user-images.githubusercontent.com/14036792/45658993-3033fa00-bac0-11e8-8a4d-4943d6b373e3.png)

This might be the only way to do it without creating the folder as "unnamed folder" which is what forces a delay on any filesystem that is slow to respond. This applies to busy disks and especially busy flash drives(see it all the time) as much as remote file systems.

**Comment by monsta** [comment]
Yeah, they don't use the editable label anymore, so the bug can't appear.
